src/serialization/IParseNode.cs

```diff
@@ -101,7 +101,7 @@ public interface IParseNode
         /// </summary>
         /// <returns>The collection of enum values.</returns>
 #if NET5_0_OR_GREATER
-        IEnumerable<T?> GetCollectionOfEnumValues<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]T>() where T : struct, Enum;
+        IEnumerable<T?> GetCollectionOfEnumValues<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields)] T>() where T : struct, Enum;
```

Comment:
Out of curiosity, do we also really need the `DynamicallyAccessedMemberTypes.NonPublicFields` or is the `DynamicallyAccessedMemberTypes.PublicFields` enough?

Reply:
Yeah, `PublicFields` should be enough